recipe_scrapers/_schemaorg.py

```diff
@@ -286,6 +286,19 @@ def ratings(self):
             return round(float(ratings), 2)
         raise SchemaOrgException("No ratingValue in SchemaOrg.")
 
+    def ratings_count(self):
+        ratings = self.data.get("aggregateRating") or self._find_entity(
+            self.data, "AggregateRating"
+        )
+        if isinstance(ratings, dict):
+            rating_id = ratings.get("@id")
+            if rating_id:
+                ratings = self.ratingsdata.get(rating_id, ratings)
+            ratings = ratings.get("ratingCount")
```

Comment:
Ok, one more question that arose as a result of reviewing #1060:

Some sites seem to use the `reviewCount` field instead of `ratingCount`; both appear to be valid according to https://schema.org documentation.

Should we retrieve from `reviewCount` as a fallback, I wonder?  I suppose in some cases reviews and ratings could be considered different..

Reply:
nice catch, guess we should considering it added ~50 additional sites to the coverage!